org.kde.kget.json

```diff
@@ -0,0 +1,140 @@
+{
+  "id" : "org.kde.kget",
+  "branch" : "master",
+  "runtime" : "org.kde.Platform",
+  "runtime-version" : "5.15-21.08",
+  "runtime-commit" : "e8e3a76d0c26375c55740919779a70e76843cca2305ec01bc3c5d3cbeae2c9b6",
+  "sdk" : "org.kde.Sdk",
+  "sdk-commit" : "5a0b113ebd0a557ee77d73107176ff5f9489da8db422a41f03e2159d43b078a8",
+  "command" : "kget",
+  "modules" : [
+    {
+      "name" : "boost",
+      "buildsystem" : "simple",
+      "sources" : [
+        {
+          "url" : "https://boostorg.jfrog.io/artifactory/main/release/1.76.0/source/boost_1_76_0.tar.bz2",
+          "sha256" : "f0397ba6e982c4450f27bf32a2a83292aba035b827a5623a14636ea583318c41",
+          "type" : "archive"
+        }
+      ],
+      "build-commands" : [
+        "./bootstrap.sh --prefix=/app --with-libraries=system",
+        "./b2 -j $FLATPAK_BUILDER_N_JOBS install"
+      ]
+    },
+    {
+      "name" : "gpgme",
+      "sources" : [
+        {
+          "url" : "https://gnupg.org/ftp/gcrypt/gpgme/gpgme-1.15.1.tar.bz2",
+          "sha256" : "eebc3c1b27f1c8979896ff361ba9bb4778b508b2496c2fc10e3775a40b1de1ad",
+          "type" : "archive"
+        }
+      ],
+      "cleanup" : [
+        "/lib/libgpgme*.la",
+        "/lib/libqgpgme.la"
+      ],
+      "modules" : [
+        {
+          "name" : "libgpg-error",
```

Comment:
libgpg-error is in the runtime

Reply:
This is indeed not needed AFAIK. See https://github.com/flathub/org.kde.kontact/pull/55/commits/07642fa2a444bdcb68cf5d1a54b038f55de7b243

Reply:
thanks for reference.